signer/keydbstore/sql_keydbstore_postgres.go

```diff
@@ -0,0 +1,24 @@
+// +build postgresqldb
```

Comment:
this is pretty ugly and is just here because of the tests. I would suggest that we have some mechanism to migrate the database outside of what GORM support (i.e. remove those sql type tags)